src/AppConfiguration.cpp

```diff
@@ -58,9 +58,14 @@ boolean AppConfiguration::readPreferences() {
     config.lightbarMaxBrightness =  doc["lightbarMaxBrightness"] | 100;
     config.lightbarTurnOffErpm =  doc["lightbarTurnOffErpm"] | 1000;
     config.ledType = doc["ledType"] | "RGB";
+    config.lightBarLedType = doc["lightBarLedType"] | "GRB";
     config.ledFrequency = doc["ledFrequency"] | "KHZ800";
+    config.lightBarLedFrequency = doc["lightBarLedFrequency"] | "KHZ800";
+    config.isLightBarReversed = doc["isLightBarReversed"] | false;
+    config.isLightBarReversed = doc["isLightBarLedTypeDifferent"] | false;
```

Comment:
config.isLightBarReversed is defined twice. Seems like C&P error.

Reply:
Thanks fixed!